doc/source/ray-core/fault_tolerance/gcs.rst

```diff
@@ -0,0 +1,69 @@
+.. _fault-tolerance-gcs:
+
+GCS Fault Tolerance
+===================
+
+Global Control Service (GCS) is a server that manages cluster-level metadata, such as locations of :ref:`actors <ray-remote-classes>`.
+It also provides a handful of cluster-level operations including :ref:`placement group <ray-placement-group-doc-ref>` scheduling and nodes management.
+By default, GCS is not fault tolerant since all the data is stored in-memory and its failure means that the entire Ray cluster fails.
+To make GCS fault tolerant, HA storage is required. Ray supports Redis as the backend storage for durability and high availability. Then, when GCS restarts, it loads all the data from the Redis instance and resumes regular functions.
+
+During the recovery period, the following functions are not available:
+
+- Actor creation, deletion and reconstruction.
+- Placement group creation, deletion and reconstruction.
+- Resource management.
+- Worker node registration.
+- Worker process creation.
+
+But the running Ray tasks and actors remain alive and any existing objects will continue to be available.
```

Comment:
If the GCS is dead for a long time, Raylet will exit. 
https://sourcegraph.com/github.com/ray-project/ray@master/-/blob/src/ray/common/ray_config_def.h?L420:21

Reply:
Yea, I mentioned later in this doc.